### PR TITLE
CDR Plus Go Live Review

### PR DESCRIPTION
**Pull request by perlboy**
Initial Go Live review merge.

### CODE REVIEW

**Comment by perlboy** [comment]
Yeah ok, so leave it in as MAY but declare it as SHALL in Sharing V1 for now. Sounds good to me, can you create the PR on Sharing V1?

**Comment by benkolera** [comment]
I think this omits the CDR restriction of `public clients MUST NOT be supported` where FAPI has them as a MAY support public clients. I'l dig to confirm and do a change.

**Comment by benkolera** [comment]
Also missing: Banning refresh token cycling. I'll add this as well.

**Comment by benkolera** [comment]
I'm not sure that this baseline is strong enough on the PAR front. FAPI 1 Adv still allows non par request but the latest statements in the CDS for request objects state:

`Authorisation request data MUST only be accepted using PAR.`

`Data Recipients MUST ONLY send authorisation request data using [[RFC9126]] and use [[PKCE]] in accordance with [[FAPI-1.0-Advanced]].`

But I do note that these transitionary FAPI migration statements were quite tricky and I misinterpreted them more than a few times. :) 

My current interpretation is that every holder CDR realm should have require_pushed_authorization_requests set to true.

File: draft-authors-cdrplus-infosec-baseline.md [hunk 1]
```diff
@@ -260,10 +275,13 @@ This document relies upon the CDR Data Standards and we therefore acknowledge th
...
 
-
 <reference anchor="FAPI-1.0-Baseline" target="https://openid.net/specs/openid-financial-api-part-1-1_0.html"> <front><title abbrev="FAPI 1.0 Baseline">Financial-grade API Security Profile 1.0 - Part 1: Baseline</title><author initials="N." surname="Sakimura" fullname="Nat Sakimura"><organization>Nat Consulting</organization></author><author initials="J." surname="Bradley" fullname="John Bradley"><organization>Yubico</organization></author><author initials="E." surname="Jay" fullname="Illumila"><organization>Illumila</organization></author></front> </reference>
 
 <reference anchor="CDRPLUS-INFOSEC-SHARING-V1" target="https://cdrplus.github.io/cdrplus-specs/main/cdrplus-infosec-sharing-v1.html"> <front><title>CDR: Sharing Arrangement V1</title><author initials="S." surname="Low" fullname="Stuart Low"><organization>Biza.io</organization></author></front> </reference>
 
+<reference anchor="CDRPLUS-ADMISSION-CONTROL" target="https://cdrplus.github.io/cdrplus-specs/main/cdrplus-admission-control.html"> <front><title>CDR: Admission Control</title><author initials="S." surname="Low" fullname="Stuart Low"><organization>Biza.io</organization></author></front> </reference>
```
Comment (benkolera) [inline draft-authors-cdrplus-infosec-baseline.md:282]: We should probably create this repo with the template and some coming soon notice.

File: draft-authors-cdrplus-infosec-baseline.md [hunk 2]
```diff
@@ -113,6 +113,8 @@ In addition, the authorisation server:
 1. **MUST** support the [@!OIDC-Core] scopes `openid` and `profile`;
 2. **SHALL** support signed ID Tokens and **MAY** support signed and encrypted ID Tokens;
 1. **SHALL** issue access tokens with an expiry time of between 2 and 10 minutes;
+1. **SHALL NOT** support public clients
```
Comment (perlboy) [inline draft-authors-cdrplus-infosec-baseline.md:116]: FAPI 1.0 Advanced states in the introduction:
> This profile does not support public clients.

I don't think it is worthwhile restating this.

File: draft-authors-cdrplus-infosec-baseline.md [hunk 3]
```diff
@@ -113,6 +113,8 @@ In addition, the authorisation server:
...
+1. **SHALL NOT** accept authorisation requests by value
```
Comment (perlboy) [inline draft-authors-cdrplus-infosec-baseline.md:117]:
```suggestion
1. **SHALL NOT** accept authorisation request object passed by value
```
Fixing the specific naming of this

**Review by perlboy — COMMENTED** [review]
@benkolera check out comments on changes.

File: draft-authors-cdrplus-infosec-baseline.md (hunk 2 above)
Comment (benkolera) [reply draft-authors-cdrplus-infosec-baseline.md:116]: Ahhhhh. My mistake. Baseline allows them but advanced doesn't. 

Yeah agreed doesn't need restating. That was pebcak sorry.

**Comment by perlboy** [comment]
Ok, this looks good now. Will go create the admission control placeholder and then merge.

File: draft-authors-cdrplus-infosec-baseline.md (hunk 1 above)
Comment (perlboy) [reply draft-authors-cdrplus-infosec-baseline.md]: Placeholder created and url updated.

**Pull request merged**